dallinger/command_line/lib/ec2.py

```diff
@@ -655,6 +668,30 @@ def _get_instance_id_from(
     )["instance_id"]
 
 
+def wait_for_instance_state_change(region, name, state, n_tries=12, wait=10):
+    with yaspin(
+        text=f"Waiting for the instance to change to state '{state}': ", color="yellow"
+    ) as sp:
+        for _ in range(n_tries):
+            instance_row = _get_instance_row_from(
+                region_name=region,
+                instance_name=name,
+                filter_by=None,
+            )
+            if instance_row["state"] == state:
+                break
+            time.sleep(wait)
+        if instance_row["state"] != state:
+            sp.fail("❌")
+            raise Exception(
+                f"Instance did not change to state '{name}' after {n_tries * wait} seconds"
```

Comment:
This should rather be 
`Instance did not change to state '{state}' after {n_tries * wait} seconds`
?

Reply:
Ups, sorry. Fixed it.